bioio/bio_image.py

```diff
@@ -196,7 +249,12 @@ def determine_plugin(
             image_str,
             msg_extra=(
                 "You may need to install an extra format dependency. "
-                "See bioio README for list of some known plugins."
+                "See plug-in registry in the bioio README Here: "
```

Comment:
I would simply change this to "See our list of known plugins in the bioio README here: ..."

Weird to say but I feel like plugin registry is a fancy term that non-software people may not know / understand the context.

Reply:
resolved in [df26332](https://github.com/bioio-devs/bioio/pull/63/commits/df2633236fef6210a3b367a6fd09248e8af6d42e)